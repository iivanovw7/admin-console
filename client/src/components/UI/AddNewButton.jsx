import React from 'react';
import PropTypes from 'prop-types';
import Fab from '@material-ui/core/Fab';
import AddIcon from '@material-ui/icons/Add';
import Button from '@material-ui/core/Button';
import withWidth from '@material-ui/core/withWidth';

const AddNewButton = props => {

  const { classes, width, history, element } = props;

  function handleClick() {
    return history.push(history.push(`/${element}/new`));
  }

  const mobileButton = () => {
    return (
      <Button size="small" className={classes.margin}
              onClick={() => {
                handleClick();
              }}>
        NEW
      </Button>
    );
  };

  const desktopButton = () => {
    return (
      <Fab color="primary" size="small" aria-label="Add" className={classes.fab}
           onClick={() => {
             handleClick();
           }}>
        <AddIcon/>
      </Fab>
    );
  };

  const components = {
    xs: mobileButton(),
    sm: mobileButton()
  };

  const Component = components[width] || desktopButton();

  return (
    <div className={classes.formRoot}>
      {Component}
    </div>
  );

};

AddNewButton.propTypes = {
  width: PropTypes.string.isRequired
};

export default withWidth()(AddNewButton);

