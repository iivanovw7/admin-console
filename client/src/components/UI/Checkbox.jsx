import Checkbox from '@material-ui/core/Checkbox';
import PropTypes from 'prop-types';
import React from 'react';

export const ActionCheckBox = props => {

  const { status, classes, handleClick } = props;

  return (
    <Checkbox
      className={classes.actionCheckBox}
      checked={status}
      size={'small'}
      color={'primary'}
      onChange={handleClick}
      disabled={false}
    />
  );

};

ActionCheckBox.propTypes = {
  classes: PropTypes.object,
  handleClick: PropTypes.func,
  status: PropTypes.bool
};
