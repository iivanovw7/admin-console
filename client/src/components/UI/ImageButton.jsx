import { Button, IconButton, ListItem } from '@material-ui/core';
import PropTypes from 'prop-types';
import React from 'react';

export const ImageButton = props => {

  const { classes, image, alt, handleClick, title } = props;

  return (
    <ListItem style={{ padding: 0, marginTop: 3, marginBottom: 3 }}>
      <IconButton className={classes.desktop} color="default" aria-label={alt} onClick={handleClick}
                  size={'small'}>
        <img style={{ width: '24px' }} src={image} alt={alt}/>
      </IconButton>
      <Button
        style={{ fontSize: '0.5rem' }}
        className={classes.mobile} size="small"
        onClick={handleClick}
      >
        {title}
      </Button>
    </ListItem>
  );
};

ImageButton.propTypes = {
  handleClick: PropTypes.func.isRequired,
  image: PropTypes.any.isRequired
};

