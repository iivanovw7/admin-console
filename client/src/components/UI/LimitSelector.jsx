import React from 'react';
import InputLabel from '@material-ui/core/InputLabel';
import FormControl from '@material-ui/core/FormControl';
import Select from '@material-ui/core/Select';

export const LimitSelector = props => {

  const { classes, limit, handleLimit, limits } = props;

  return (
    <div className={classes.fromLimitSelector}>
      <FormControl className={classes.formControl}>
        <InputLabel>Results limit</InputLabel>
        <Select
          native
          value={limit}
          onChange={handleLimit}
        >
          {limits.map(newLimit => (
            <option key={newLimit} value={newLimit}>
              {newLimit}
            </option>
          ))}
        </Select>
      </FormControl>
    </div>
  );

};

