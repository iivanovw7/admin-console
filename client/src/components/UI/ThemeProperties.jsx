import blue from '@material-ui/core/colors/blue';
import { fade } from '@material-ui/core/styles/colorManipulator';
import { createMuiTheme } from '@material-ui/core/styles';

const drawerWidth = 240;

//Exporting material-ui main themeProperties configuration
export const themeProperties = createMuiTheme({
  typography: {
    useNextVariants: true
  },
  mixins: {
    toolbar: {
      textAlign: 'center',
      minHeight: '64px',
      justifyContent: 'space-between'
    }
  },
  palette: {
    primary: blue,
    secondary: {
      main: '#ffffff'
    }
  },
  status: {
    danger: 'orange'
  }
});

//Exporting material-ui main styles
export const NavigationStyles = theme => ({
  active: {
    backgroundColor: 'rgba(0, 0, 0, 0.10)'
  },
  root: {
    display: 'flex'
  },
  drawer: {
    [theme.breakpoints.up('sm')]: {
      width: drawerWidth,
      flexShrink: 0
    }
  },
  appBar: {
    marginLeft: drawerWidth,
    [theme.breakpoints.up('sm')]: {
      width: `calc(100% - ${drawerWidth}px)`
    }
  },
  menuButton: {
    marginRight: 20,
    [theme.breakpoints.up('sm')]: {
      display: 'none'
    }
  },
  toolbar: theme.mixins.toolbar,
  toolbarUserName: {
    [theme.breakpoints.down('xs')]: {
      display: 'none'
    }
  },
  drawerPaper: {
    width: drawerWidth
  },
  content: {
    flexGrow: 1,
    padding: theme.spacing.unit * 3
  },
  search: {
    position: 'relative',
    borderRadius: theme.shape.borderRadius,
    backgroundColor: fade(theme.palette.common.white, 0.15),
    '&:hover': {
      backgroundColor: fade(theme.palette.common.white, 0.25)
    },
    marginRight: theme.spacing.unit * 2,
    marginLeft: 0,
    width: '100%',
    [theme.breakpoints.up('sm')]: {
      marginLeft: theme.spacing.unit * 3,
      width: 'auto'
    }
  },
  searchIcon: {
    width: theme.spacing.unit * 9,
    height: '100%',
    position: 'absolute',
    pointerEvents: 'none',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center'
  },
  inputRoot: {
    color: 'inherit',
    width: '100%'
  },
  inputInput: {
    paddingTop: theme.spacing.unit,
    paddingRight: theme.spacing.unit,
    paddingBottom: theme.spacing.unit,
    paddingLeft: theme.spacing.unit * 10,
    transition: theme.transitions.create('width'),
    width: '100%',
    [theme.breakpoints.up('md')]: {
      width: 200
    }
  }
});

export const LoginFormStyles = theme => ({

  [theme.breakpoints.down('sm')]: {
    loginFormContainer: {
      boxShadow: 'none',
      height: '100%',
      display: 'flex',
      flexDirection: 'column',
      justifyContent: 'center'
    }
  },
  loginWrapper: {
    maxWidth: '500px',
    margin: '0 auto',
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'center',
    height: '100%'
  },
  loginFormContainer: {
    padding: theme.spacing.unit,
    maxWidth: '500px'
  },
  formPaper: {
    margin: theme.spacing.unit * 2
  },
  paddingTop: theme.spacing.unit * 2,
  paddingBottom: theme.spacing.unit * 2

});

//Exporting material-ui main styles
export const Wrapper = theme => ({
  root: {
    display: 'flex'
  },
  toolbar: theme.mixins.toolbar,
  contentSingle: {
    flexGrow: 1,
    padding: theme.spacing.unit * 3,
    marginTop: '64px',
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'space-between',
    height: '100%',
  },
  contentList: {
    flexGrow: 1,
    padding: theme.spacing.unit * 3,
    marginTop: '64px',
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'space-between',
    [theme.breakpoints.up('sm')]: {
      height: '90vh'
    },
    [theme.breakpoints.down('sm')]: {
      height: '100%',
    }
  },
  formRoot: {
    display: 'flex',
    flexWrap: 'wrap',

    buttonWrapper: {
      display: 'flex',
      alignItems: 'center'
    }
  },
  fromLimitSelector: {
    display: 'flex',
    flexWrap: 'wrap',
    [theme.breakpoints.down('sm')]: {
      display: 'none'
    }
  },
  formControl: {
    margin: theme.spacing.unit,
    minWidth: 120,
    maxWidth: 300
  },
  controlsContainer: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'space-between',
    paddingRight: 10,
    paddingLeft: 10
  },
  selectorsContainer: {
    display: 'flex',
    flexDirection: 'row',
    alignItems: 'center',
    paddingRight: 10,
    paddingLeft: 10
  }
});

export const Container = theme => ({
  root: {
    width: '100%',
    marginTop: theme.spacing.unit * 3,
    overflowX: 'auto',
    display: 'flex',
    justifyContent: 'center',
    overflow: 'hidden'
  },
  tables: {
    minWidth: 200,
    width: '100%'
  },
  tableCell: {
    paddingLeft: 10,
    paddingTop: 5,
    paddingBottom: 5
  },
  rowClass: {
    '&:hover': {
      backgroundColor: fade(theme.palette.common.black, 0.1),
      cursor: 'pointer'
    },
    [theme.breakpoints.up('sm')]: {
      height: '48px',
    },
    [theme.breakpoints.down('sm')]: {
      height: '100%',
    },
  },
  nameCell: {
    paddingRight: 5,
    paddingLeft: 10,
    paddingTop: 5,
    paddingBottom: 5,

    [theme.breakpoints.up('sm')]: {
      width: '30%',
    },

    [theme.breakpoints.down('sm')]: {
      width: '60%',
    }

  },
  addressCell: {
    paddingRight: 5,
    paddingLeft: 10,
    paddingTop: 5,
    paddingBottom: 5,
    width: '100%',
    [theme.breakpoints.down('sm')]: {
      display: 'none'
    }
  },
  employeesCell: {
    paddingRight: 5,
    paddingLeft: 10,
    paddingTop: 5,
    paddingBottom: 5,
    width: '30%',
    [theme.breakpoints.down('sm')]: {
      display: 'none'
    }
  },
  branchPaper: {
    width: '100%',
    maxWidth: 600,
    margin: theme.spacing.unit * 2
  }
});

export const Notification = theme => ({
  close: {
    padding: theme.spacing.unit / 2
  }
});

export const Buttons = theme => ({
  formRoot: {
    display: 'flex',
    flexWrap: 'wrap'
  },
  [theme.breakpoints.down('sm')]: {
    mobile: {
      display: 'flex'
    },
    desktop: {
      display: 'none'
    }
  },
  [theme.breakpoints.up('sm')]: {
    mobile: {
      display: 'none'
    },
    desktop: {
      display: 'flex'
    }
  }
});

