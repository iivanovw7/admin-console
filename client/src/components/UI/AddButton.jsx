import { withStyles } from '@material-ui/core';
import React from 'react';
import PropTypes from 'prop-types';
import Fab from '@material-ui/core/Fab';
import AddIcon from '@material-ui/icons/Add';
import Button from '@material-ui/core/Button';
import { Buttons } from './ThemeProperties';

const AddButton = props => {

  const { classes, history, element } = props;

  function handleClick() {
    return history.push(history.push(`/${element}/new`));
  }

  return (
    <div className={classes.formRoot}>
      <Button size="small" className={classes.mobile}
              onClick={() => {
                handleClick();
              }}>
        <strong>Create</strong>
      </Button>
      <Fab color="primary" size="small" aria-label="Add" className={classes.desktop}
           onClick={() => {
             handleClick();
           }}>
        <AddIcon/>
      </Fab>
    </div>
  );

};

AddButton.propTypes = {
  classes: PropTypes.object.isRequired
};

export default withStyles(Buttons, { withTheme: true })(AddButton);