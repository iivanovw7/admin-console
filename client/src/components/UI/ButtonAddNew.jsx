import { Button, Fab } from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles';
import AddIcon from '@material-ui/icons/Add';
import PropTypes from 'prop-types';
import React from 'react';
import { Buttons } from './ThemeProperties';

const ButtonAddNew = props => {

  const { classes, history, element } = props;

  function handleClick() {
    return history.push(history.push(`/${element}/new`));
  }

  return (
    <div className={classes.formRoot}>
      <Button
        size="small"
        className={classes.mobile}
        onClick={() => {
          handleClick();
        }}
      >
        <strong>Create</strong>
      </Button>
      <Fab
        color="primary"
        size="small"
        aria-label="Add"
        className={classes.desktop}
        onClick={() => {
          handleClick();
        }}>
        <AddIcon/>
      </Fab>
    </div>
  );

};

ButtonAddNew.propTypes = {
  classes: PropTypes.object.isRequired
};

export default withStyles(Buttons, { withTheme: true })(ButtonAddNew);