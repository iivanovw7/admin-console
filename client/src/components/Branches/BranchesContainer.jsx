import PropTypes from 'prop-types';
import React from 'react';
import { connect } from 'react-redux';
import Paper from '@material-ui/core/Paper';
import Table from '@material-ui/core/Table';
import TableHead from '@material-ui/core/TableHead';
import TableBody from '@material-ui/core/TableBody';
import TableCell from '@material-ui/core/TableCell';
import TableRow from '@material-ui/core/TableRow';
import { withStyles } from '@material-ui/core/styles';
import { Branches } from '../UI/ThemeProperties';
import { fetchBranch } from '../../actions/branches';
import { withRouter } from 'react-router-dom';

const BranchesContainer = props => {
  const { classes, history, dispatch } = props;

  function displayStatus(status) {
    return (status) ? ('Active') : ('Disabled');
  }

  function handleBranchClick(id) {
    dispatch(fetchBranch(id, history));
  }

  return (
    <Paper className={classes.root}>
      <Table className={classes.table}>
        <TableHead>
          <TableRow>
            <TableCell className={classes.nameCell}>Name</TableCell>
            <TableCell className={classes.addressCell} align="center">Address</TableCell>
            <TableCell className={classes.tableCell} align="center">Employees</TableCell>
            <TableCell className={classes.tableCell} align="center">Status</TableCell>
          </TableRow>
        </TableHead>
        <TableBody>
          {props.branches.list.output.map(row => (
            <TableRow key={row._id} onClick={() => {
              handleBranchClick(row._id);
            }} className={classes.rowClass}>
              <TableCell component="th" scope="row" className={classes.nameCell}>
                {row.name}
              </TableCell>
              <TableCell className={classes.addressCell} align="center">{row.address}</TableCell>
              <TableCell className={classes.tableCell} align="center">xxx</TableCell>
              <TableCell className={classes.tableCell}
                         align="center">{displayStatus(row.status)}</TableCell>
            </TableRow>
          ))}
        </TableBody>
      </Table>
    </Paper>
  );
};

BranchesContainer.propTypes = {
  classes: PropTypes.object.isRequired
};

function mapStateToProps(state) {
  return { branches: state.branches };
}

export default connect(mapStateToProps, { fetchBranch })(withStyles(Branches)(withRouter(BranchesContainer)));

