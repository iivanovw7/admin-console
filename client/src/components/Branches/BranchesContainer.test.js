import { withStyles } from '@material-ui/core';
import { createMount, createShallow } from '@material-ui/core/test-utils';
import React from 'react';
import { connect, Provider } from 'react-redux';
import { MemoryRouter, withRouter } from 'react-router-dom';
import * as mocks from '../../__mocks__/';
import { getSingleBranch } from '../../actions';
import BranchesContainer from '../../components/Branches/BranchesContainer';
import { Container } from '../../components/UI/ThemeProperties';
import { history, storeFactory } from '../../testUtils';

const setup = (initialState = {}) => {
  let wrapper;
  let shallowWithDive;
  let mount;
  let shallow;
  let store = storeFactory(initialState);

  const Composition = connect(
    null, { getSingleBranch }
  )(withStyles(Container, { withTheme: true })(withRouter(BranchesContainer)));

  const props = {
    branches: mocks.BRANCHES.output,
    history: history
  };

  beforeEach(() => {
    shallow = createShallow();
    shallowWithDive = createShallow({ dive: true });
    mount = createMount();
  });

  afterEach(() => {
    mount.cleanUp();
  });

  /**
   *  Renders branches container with styles
   */
  describe('Branches container', () => {
    it('Should render branches container with fields and styles', () => {
      wrapper = mount(
        <Provider store={store}>
          <MemoryRouter>
            <Composition {...props} />
          </MemoryRouter>
        </Provider>
      );
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
};

setup();
