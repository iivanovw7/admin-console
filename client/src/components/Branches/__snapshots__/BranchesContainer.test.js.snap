// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Branches container Should render branches container with fields and styles 1`] = `"<div class=\\"MuiPaper-root-59 MuiPaper-elevation2-63 MuiPaper-rounded-60 withRouter-BranchesContainer--root-30 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----root-1\\"><table class=\\"MuiTable-root-86 withRouter-BranchesContainer--tables-38 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----tables-9\\"><thead class=\\"MuiTableHead-root-87\\"><tr class=\\"MuiTableRow-root-88 MuiTableRow-head-91\\"><th class=\\"MuiTableCell-root-93 MuiTableCell-head-94 withRouter-BranchesContainer--branchNameCell-41 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchNameCell-12\\" scope=\\"col\\"><h3>Name</h3></th><th class=\\"MuiTableCell-root-93 MuiTableCell-head-94 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchAddressCell-42 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchAddressCell-13\\" scope=\\"col\\"><h3>Address</h3></th><th class=\\"MuiTableCell-root-93 MuiTableCell-head-94 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchEmployeesCell-43 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchEmployeesCell-14\\" scope=\\"col\\"><h3>Employees</h3></th><th class=\\"MuiTableCell-root-93 MuiTableCell-head-94 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--tableCell-39 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----tableCell-10\\" scope=\\"col\\"><h3>Status</h3></th></tr></thead><tbody class=\\"MuiTableBody-root-105\\"><tr class=\\"MuiTableRow-root-88 withRouter-BranchesContainer--branchRowClass-40 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchRowClass-11\\" style=\\"opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;\\"><th class=\\"MuiTableCell-root-93 MuiTableCell-body-95 withRouter-BranchesContainer--branchNameCell-41 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchNameCell-12\\" scope=\\"row\\">San Mart√≠n de los Andes</th><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchAddressCell-42 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchAddressCell-13\\">938 Barby Way</td><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchEmployeesCell-43 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchEmployeesCell-14\\"><div class=\\"MuiCircularProgress-root-106 MuiCircularProgress-colorPrimary-109 MuiCircularProgress-indeterminate-108\\" style=\\"width: 12px; height: 12px;\\" role=\\"progressbar\\"><svg class=\\"MuiCircularProgress-svg-111\\" viewBox=\\"22 22 44 44\\"><circle class=\\"MuiCircularProgress-circle-112 MuiCircularProgress-circleIndeterminate-114\\" cx=\\"44\\" cy=\\"44\\" r=\\"20.2\\" fill=\\"none\\" stroke-width=\\"3.6\\"></circle></svg></div></td><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--tableCell-39 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----tableCell-10\\"><strong>Active</strong></td></tr><tr class=\\"MuiTableRow-root-88 withRouter-BranchesContainer--branchRowClass-40 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchRowClass-11\\" style=\\"opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;\\"><th class=\\"MuiTableCell-root-93 MuiTableCell-body-95 withRouter-BranchesContainer--branchNameCell-41 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchNameCell-12\\" scope=\\"row\\">Dingzhai</th><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchAddressCell-42 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchAddressCell-13\\">903 Dottie Street</td><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchEmployeesCell-43 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchEmployeesCell-14\\"><div class=\\"MuiCircularProgress-root-106 MuiCircularProgress-colorPrimary-109 MuiCircularProgress-indeterminate-108\\" style=\\"width: 12px; height: 12px;\\" role=\\"progressbar\\"><svg class=\\"MuiCircularProgress-svg-111\\" viewBox=\\"22 22 44 44\\"><circle class=\\"MuiCircularProgress-circle-112 MuiCircularProgress-circleIndeterminate-114\\" cx=\\"44\\" cy=\\"44\\" r=\\"20.2\\" fill=\\"none\\" stroke-width=\\"3.6\\"></circle></svg></div></td><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--tableCell-39 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----tableCell-10\\"><strong>Active</strong></td></tr><tr class=\\"MuiTableRow-root-88 withRouter-BranchesContainer--branchRowClass-40 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchRowClass-11\\" style=\\"opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;\\"><th class=\\"MuiTableCell-root-93 MuiTableCell-body-95 withRouter-BranchesContainer--branchNameCell-41 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchNameCell-12\\" scope=\\"row\\">Cantao-an</th><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchAddressCell-42 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchAddressCell-13\\">7827 Luster Point</td><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--branchEmployeesCell-43 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----branchEmployeesCell-14\\"><div class=\\"MuiCircularProgress-root-106 MuiCircularProgress-colorPrimary-109 MuiCircularProgress-indeterminate-108\\" style=\\"width: 12px; height: 12px;\\" role=\\"progressbar\\"><svg class=\\"MuiCircularProgress-svg-111\\" viewBox=\\"22 22 44 44\\"><circle class=\\"MuiCircularProgress-circle-112 MuiCircularProgress-circleIndeterminate-114\\" cx=\\"44\\" cy=\\"44\\" r=\\"20.2\\" fill=\\"none\\" stroke-width=\\"3.6\\"></circle></svg></div></td><td class=\\"MuiTableCell-root-93 MuiTableCell-body-95 MuiTableCell-alignCenter-102 withRouter-BranchesContainer--tableCell-39 withRouter-Connect-WithStyles-withRouter-BranchesContainer-----tableCell-10\\"><strong>Disabled</strong></td></tr></tbody></table></div>"`;
